ASAManager/ASAManager.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.0-preview-20180307-01" />
```

Comment:
any reason for using a preview version rather than a stable one? could this lead to flaky tests?

Reply:
I had copied what IotHubManagerTests was using. Looks like PcsConfig also uses the preview, storage uses 15.6.1. I don't have a strong opinion either way, but we should probably be consistent.